packages/backend/src/apps/custom-api/actions/http-request/index.ts

```diff
@@ -61,13 +61,15 @@ const action: IRawAction = {
           type: 'string' as const,
           required: true,
           variables: false,
+          customStyle: { flex: 2 },
         },
         {
           placeholder: 'Value',
           key: 'value',
           type: 'string' as const,
           required: true,
           variables: true,
+          customStyle: { flex: 3 },
```

Comment:
i think need to add `minWidth: 0` or else this happens
<img width="842" alt="image" src="https://github.com/user-attachments/assets/b025b9f9-56bd-4a68-a8c0-e1d2512a2073" />